src/graphql/main/schema.graphql

```diff
@@ -309,6 +314,11 @@ type Globals {
   This can be used to know if an invoice belongs to one of our nodes.
   """
   nodesIds: [String!]!
+
+  """
+  A list of supported CCA2 country codes for this Galoy instance
+  """
+  supportedCountries: [CountryCode!]!
```

Comment:
uhmm now I see why you wanted unsupported countries. if you only return country code you still need to use another source to render the component... alternative is to create a country entity with all the needed data (country code as id, prefix, flag, name).. not sure if the libphonelibrary has the fields but there are other sources too https://github.com/LucianoGanga/country-codes-list/blob/master/countriesData.js

Reply:
We already use another source to render the phone code picker, so it is not a big deal.

Reply:
uhm I guess this requires more review then @bodymindarts could you please validate the schema?... we will have a breaking change if we decide to add different providers for country.. at least we need to have a `Country` type (even if it only has countryCode attr)